### PR TITLE
[onert] bind nnfw.h with python

### PR DESCRIPTION
**Pull request by blue4683**
This commit binds nnfw.h with python
Added source code and header file related to nnfw_api_pybind

ONE-DCO-1.0-Signed-off-by: blue4683 <an4683@gmail.com>

Related #11487

### CODE REVIEW

**Comment by ragmani** [comment]
@nnfw-bot test onert-x64-release

**Comment by ragmani** [comment]
@nnfw-bot test onert-x64-release

File: runtime/onert/python/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+add_subdirectory(api)
```
Comment (ragmani) [inline runtime/onert/python/CMakeLists.txt:1]:
```suggestion
if(NOT BUILD_PYTHON_BINDING)
  return()
endif(NOT BUILD_PYTHON_BINDING)

add_subdirectory(api)
```
with the patch as follows:
```patch
diff --git a/infra/nnfw/cmake/options/options_aarch64-android.cmake b/infra/nnfw/cmake/options/options_aarch64-android.cmake
index 5de2be333..a1d802c60 100644
--- a/infra/nnfw/cmake/options/options_aarch64-android.cmake
+++ b/infra/nnfw/cmake/options/options_aarch64-android.cmake
@@ -11,8 +11,11 @@ option(DOWNLOAD_OPENGL_HEADERS "Download Opengl_headers source" ON)
 option(DOWNLOAD_EGL_HEADERS "Download Egl_headers source" ON)
 option(DOWNLOAD_VULKAN "Download vulkan source" ON)
 option(DOWNLOAD_OPENCL_HEADERS "Download Opencl_headers source" ON)
-option(DOWNLOAD_PYBIND11 "Download Pybind11 source" ON)
+
 option(BUILD_GPU_CL "Build gpu_cl backend" ON)
 option(BUILD_TENSORFLOW_LITE_GPU "Build TensorFlow Lite GPU delegate from the downloaded source" ON)
 
 option(BUILD_MINMAX_H5DUMPER "Build minmax h5dumper" OFF)
+
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" OFF)
+option(BUILD_PYTHON_BINDING "Build python binding" OFF)
diff --git a/infra/nnfw/cmake/options/options_aarch64-tizen.cmake b/infra/nnfw/cmake/options/options_aarch64-tizen.cmake
index cccd77f98..8ebec4183 100644
--- a/infra/nnfw/cmake/options/options_aarch64-tizen.cmake
+++ b/infra/nnfw/cmake/options/options_aarch64-tizen.cmake
@@ -12,6 +12,10 @@ option(GENERATE_RUNTIME_NNAPI_TESTS "Generate NNAPI operation gtest" OFF)
 option(ENVVAR_ONERT_CONFIG "Use environment variable for onert configuration" OFF)
 
 option(BUILD_NPUD "Build NPU daemon" ON)
+
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" OFF)
+option(BUILD_PYTHON_BINDING "Build python binding" OFF)
+
 # Do not allow to use CONFIG option on Tizen
 option(ENVVAR_NPUD_CONFIG "Use environment variable for npud configuration" OFF)
 
diff --git a/infra/nnfw/cmake/options/options_armv7hl-tizen.cmake b/infra/nnfw/cmake/options/options_armv7hl-tizen.cmake
index 07dc0404e..d73736025 100644
--- a/infra/nnfw/cmake/options/options_armv7hl-tizen.cmake
+++ b/infra/nnfw/cmake/options/options_armv7hl-tizen.cmake
@@ -20,6 +20,10 @@ option(BUILD_GPU_CL "Build gpu_cl backend" ON)
 option(BUILD_TENSORFLOW_LITE_GPU "Build TensorFlow Lite GPU delegate from the downloaded source" ON)
 
 option(BUILD_NPUD "Build NPU daemon" ON)
+
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" OFF)
+option(BUILD_PYTHON_BINDING "Build python binding" OFF)
+
 # Do not allow to use CONFIG option on Tizen
 option(ENVVAR_NPUD_CONFIG "Use environment variable for npud configuration" OFF)
 
diff --git a/infra/nnfw/cmake/options/options_armv7l-tizen.cmake b/infra/nnfw/cmake/options/options_armv7l-tizen.cmake
index 4fdcbc33f..fb8d64e12 100644
--- a/infra/nnfw/cmake/options/options_armv7l-tizen.cmake
+++ b/infra/nnfw/cmake/options/options_armv7l-tizen.cmake
@@ -20,6 +20,10 @@ option(BUILD_GPU_CL "Build gpu_cl backend" ON)
 option(BUILD_TENSORFLOW_LITE_GPU "Build TensorFlow Lite GPU delegate from the downloaded source" ON)
 
 option(BUILD_NPUD "Build NPU daemon" ON)
+
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" OFF)
+option(BUILD_PYTHON_BINDING "Build python binding" OFF)
+
 # Do not allow to use CONFIG option on Tizen
 option(ENVVAR_NPUD_CONFIG "Use environment variable for npud configuration" OFF)
 
diff --git a/infra/nnfw/cmake/options/options_i686-tizen.cmake b/infra/nnfw/cmake/options/options_i686-tizen.cmake
index e51f5ed7b..1d690b2cb 100644
--- a/infra/nnfw/cmake/options/options_i686-tizen.cmake
+++ b/infra/nnfw/cmake/options/options_i686-tizen.cmake
@@ -12,5 +12,8 @@ option(ENVVAR_ONERT_CONFIG "Use environment variable for onert configuration" OF
 
 option(BUILD_XNNPACK "Build XNNPACK" OFF)
 
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" OFF)
+option(BUILD_PYTHON_BINDING "Build python binding" OFF)
+
 option(BUILD_MINMAX_H5DUMPER "Build minmax h5dumper" OFF)
 option(ENABLE_ONERT_TRAIN "Enable onert training feature" OFF)
diff --git a/infra/nnfw/cmake/options/options_x86_64-tizen.cmake b/infra/nnfw/cmake/options/options_x86_64-tizen.cmake
index 70da68c82..53cb1832b 100644
--- a/infra/nnfw/cmake/options/options_x86_64-tizen.cmake
+++ b/infra/nnfw/cmake/options/options_x86_64-tizen.cmake
@@ -14,6 +14,10 @@ option(BUILD_XNNPACK "Build XNNPACK" OFF)
 option(DOWNLOAD_OPENCL_HEADERS "Download opencl headers" OFF)
 
 option(BUILD_NPUD "Build NPU daemon" ON)
+
+option(DOWNLOAD_PYBIND11 "Download Pybind11 source" OFF)
+option(BUILD_PYTHON_BINDING "Build python binding" OFF)
+
 # Do not allow to use CONFIG option on Tizen
 option(ENVVAR_NPUD_CONFIG "Use environment variable for npud configuration" OFF)
```

**Comment by jaemaning** [comment]
@nnfw-bot test

**Comment by blue4683** [comment]
@nnfw-bot test onert-x64-release

**Comment by blue4683** [comment]
@nnfw-bot test onert-x64-release

**Comment by blue4683** [comment]
@nnfw-bot test onert-cross-debug

**Comment by ragmani** [comment]
@nnfw-bot test onert-cross-release

**Comment by ragmani** [comment]
@nnfw-bot test onert-cross-debug

**Pull request closed without merging**